internal/printer/printer.go

```diff
@@ -3480,6 +3485,21 @@ func (p *Printer) emitTypeAliasDeclaration(node *ast.TypeAliasDeclaration) {
 	p.exitNode(node.AsNode(), state)
 }
 
+func (p *Printer) emitJSTypeAliasDeclaration(node *ast.JSTypeAliasDeclaration) {
+	state := p.enterNode(node.AsNode())
+	p.emitModifierList(node.AsNode(), node.Modifiers(), false /*allowDecorators*/)
+	p.writeKeyword("jstype")
```

Comment:
Is this just temporary?

Reply:
No, although it might be wrong. At least, I think the printer needs to be able to print every single node. Whether it should use a made-up keyword to indicate that it's reparsed, I don't know. Maybe it should emit `type` instead.